AutoClicker/Utils/GloablMouseHook.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Drawing;
+using Serilog;
+
+namespace AutoClicker.Utils
+{
+
+    /// <summary>
+    /// This class has been copied from StackOverflow where the OP copied it form somewhere else...
+    /// </summary>
+    public static class GloablMouseHook
```

Comment:
You have a typo here, should be "GlobalMouseHook"

Reply:
I'll fix the typos later today. Thanks for the comments.